docs/AutomationReference.md

```diff
@@ -280,12 +314,25 @@ These versions embed the symbols into the assemblies and support [SourceLink](ht
 
 #### Error handling
 
-`AxeWindowsAutomationException` is thrown for all unhandled errors in Axe.Windows.Automation. If an exception was thrown from code not owned by Axe.Windows.Automation, that exception will be wrapped in the `Exception.InnerException` property.
+`AxeWindowsAutomationException` is thrown for errors in `Axe.Windows.Automation`'s logic where meaningful error reporting can be generated. In some situations, the `Exception.InnerException` property may contain a corresponding system-level exception for errors encountered by Axe.Windows. Other exceptions may be thrown by the system from calls to this library.
```

Comment:
Can we add a note here that if the cancellation token is cancelled, then we will also throw an aggregate exception which contains a `TaskCancelledException`?